docs/dr.md

```diff
@@ -465,6 +466,284 @@ This log includes detailed information that may help to troubleshoot the
 validate application command. If the command failed, check the error details in
 the log.
 
+### validate clusters
+
+The validate clusters command validates the disaster recovery clusters by
+gathering cluster scoped and related ramen resources from all clusters.
+
+### Validating clusters
+
+To validate the disaster recovery clusters, run the following command:
+
+```console
+$ odf dr validate clusters -o out
+⭐ Using config "config.yaml"
+⭐ Using report "out"
+
+🔎 Validate config ...
+   ✅ Config validated
+
+🔎 Validate clusters ...
+   ✅ Gathered data from cluster "hub"
+   ✅ Gathered data from cluster "dr1"
+   ✅ Gathered data from cluster "dr2"
+   ✅ Clusters validated
+
+✅ Validation completed (36 ok, 0 stale, 0 problem)
+```
+
+The command gathered cluster scoped and ramen resources from all clusters,
+inspected the resources, and stored output files in the specified output
+directory:
+
+```console
+$ tree -L1 out
+out
+├── validate-clusters.data
+├── validate-clusters.log
+└── validate-clusters.yaml
+```
+
+> [!IMPORTANT]
+> When reporting DR related issues, please create an archive with the output
+> directory and upload it to the issue tracker.
+
+#### The validate-clusters.yaml
+
+The `validate-clusters.yaml` report is a machine and human readable description
+of the command and the clusters status.
+
+The most important part of the report is the clustersStatus:
```

Comment:
```suggestion
The most important part of the report is the clusters status:
```
?

Reply:
Pushing an update for this and applicationStatus...